joomla/joomla-cms
# [#32833] Implements a translatable language constant for the InvalidArgumentException in com_ajax

Tracker item at http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=32833

To test
- Install https://github.com/Joomla-Ajax-Interface/Ajax-Session-Module/archive/master.zip
- Navigate to `?option=com_ajax&module=session&cmd=add&data=foo` and observe the error message
- Change `COM_AJAX_SPECIFY_FORMAT` /language/en-GB/en-GB.com_ajax.ini or /administrator/language/en-GB/en-GB.com_ajax.ini to some other message
- Navigate to `?option=com_ajax&module=session&cmd=add&data=foo` and observe the revised error message
- Navigate to `?option=com_ajax&module=bob&format=debug`
- Navigate to `?option=com_ajax&module=session&format=debug&method=foo`
- Then rename `/modules/mod_session/helper.php` to `/modules/mod_session/~helper.php` and then navigate to `?option=com_ajax&module=session&format=debug`
